appsettings.json

```diff
@@ -9,6 +9,13 @@
       "Microsoft.Hosting.Lifetime": "Information"
     }
   },
-  "AllowedHosts": "*",
-  "url": "https://cuttlefish.azurewebsites.net"
-}
\ No newline at end of file
+  "EmailSettings": {
+    "From": "cuttlefishcapstoneteam@gmail.com",
+    "SmtpServer": "smtp.gmail.com",
+    "Port": 465,
+    "Username": "CuttlefishAdmin",
+    "Password": ""
```

Comment:
Are we gonna need to set up a repo secret for this?

Reply:
yes